src/test/java/org/cactoos/io/TeeOutputTest.java

```diff
@@ -171,43 +152,34 @@ public void copiesWithFile() throws Exception {
                     )
                 )
             ),
-            new TextHasString(
-                new MatcherOf<>(
-                    str -> {
-                        return new String(
-                            new TextOf(file.toPath()).asString()
-                        ).equals(str);
-                    }
-                )
+            new TextIs(
+                new TextOf(file.toPath())
             )
         );
     }
 
     @Test
     public void copiesWithOutputStream() {
         final ByteArrayOutputStream baos = new ByteArrayOutputStream();
-        final ByteArrayOutputStream copy = new ByteArrayOutputStream();
-        MatcherAssert.assertThat(
+        new Assertion<>(
             "Can't copy Output with output stream",
-            new TextOf(
-                new TeeInput(
-                    new InputOf("Hello, товарищ! with output stream"),
-                    new TeeOutput(
-                        new OutputTo(baos),
-                        copy
+            () -> new TextOf(
+                new Sticky(
```

Comment:
@Iprogrammerr Is necessary use the Sticky decorator here?

Reply:
@fabriciofx Same as above.

Reply:
@Iprogrammerr the same above.